### PR TITLE
Removed banners mentions from translations

### PR DESCRIPTION
**Pull request by dedepete**
Since banners were not introduced into release, these mentions are out of any usage.

### CODE REVIEW

**Comment by k0shk0sh** [comment]
@dedepete please remove the text and I'll push to this branch later to remove the related stuff to it.

**Comment by dedepete** [comment]
@k0shk0sh reverted last two commits.

**Comment by dedepete** [comment]
@k0shk0sh @yakov116 All the files related to banners have been removed. Please, check if everything is alright.

File: app/src/main/java/com/fastaccess/ui/modules/repos/wiki/WikiActivity.kt [hunk 1]
```diff
@@ -119,7 +119,7 @@ class WikiActivity : BaseActivity<WikiMvp.View, WikiPresenter>(), WikiMvp.View {
                 return true
             }
             R.id.share -> {
-                ActivityHelper.shareUrl(this, "${LinkParserHelper.PROTOCOL_HTTPS}://${LinkParserHelper.HOST_DEFAULT}" +
+                ActivityHelper.shareUrl(this, "${LinkParserHelper.PROTOCOL_HTTPS}://${LinkParserHelper.HOST_DEFAULT}/" +
```
Comment (k0shk0sh) [inline app/src/main/java/com/fastaccess/ui/modules/repos/wiki/WikiActivity.kt]: I had rather open new pr for this instead so we get better Changelog :)

**Comment by dedepete** [comment]
@k0shk0sh removed that commit. Will open a new PR then.

**Pull request merged**